### ISSUE TITLE
Color text should be white or black depending on color contrast

### ISSUE THREAD

**Issue by tkbartel** [tkbartel/seasons#5]
For example, "Black Beauty" text should render as white

<img width="465" alt="Image" src="https://github.com/user-attachments/assets/a65e6fc1-bf56-42c4-a6ac-0ef596c8e4c5" />

This only applies to complementary and neutral colors, as the accent color text is not overlaid on the color square.
